iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -115,20 +115,20 @@ class RouDi
                          const uint64_t sessionId,
                          const version::VersionInfo& versionInfo);
 
-    /// @brief Creates a unique ID which can be used to check outdated mqueue transmissions
+    /// @brief Creates a unique ID which can be used to check outdated IPC channel transmissions
     /// @return a unique, monotonic and consecutive increasing number
     static uint64_t getUniqueSessionIdForProcess();
 
   private:
-    void mqThread();
+    void ipcChannelThread();
```

Comment:
We should use the chance to clear up the naming a bit. ipcChannelThread does not really reflect what's happenning in there. Basically we are retrieving ipc channel messages from the runtime and forward them to `processMessage`.
Possible name:
`retrieveAndProcessRuntimeMessageThread`

Reply:
in my previous life, we called this stuff `CmdHandler` ... maybe I'm getting old :D

Reply:
Good point, renamed `processThread()` accordingly.